### DISCUSSION TITLE
Larger Input Data Set

### DISCUSSION THREAD

**Discussion by asim215** [binarybottle/engram-layout#79] [Ideas]
Hello. I curious to ask following.
If we put much larger data sets as input to train model, is it possible to get better final result?
For example, more code (github), books (gutenberg).

**Comment by binarybottle** [comment]
Please see https://github.com/binarybottle/engram-2021/discussions/86#discussion-10126586 for an announcement of the new Engram approach/layouts/repo! I think you will appreciate the data-driven approach.
